android/java/apk_for_test.flags

```diff
@@ -884,3 +884,7 @@
 -keep class org.chromium.chrome.browser.omnibox.styles.OmniboxResourceProvider {
     *** getToolbarSidePaddingForNtp(...);
 }
+
+-keep class org.chromium.chrome.browser.tasks.tab_management.TabSwitcherPaneBase {
```

Comment:
This is excessive since `android/java/proguard.flags` used for tests as well

Reply:
thanks, removed at https://github.com/brave/brave-core/pull/25130/commits/58f3bb454b7573de94ae9c05ffd7908dba516fc5#diff-328021dc35ff453fa79b8f57c279b5365d6c1a095a10ebd8bb8f98a67cdd8865L888